account_invoice_under_payment/views/payment_line_views.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="payment_order_view_under_payment_form" model="ir.ui.view">
+            <field name="name">paymentorder.under_payment_form</field>
+            <field name="model">payment.order</field>
+            <field name="inherit_id" ref="account_payment.view_payment_order_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='move_line_id']" position="after">
+                    <field name="failed"/>
+                </xpath>
+                <xpath expr="//field[@name='line_ids']/tree/field[@name='name']" position="after">
+                    <field name="failed"/>
```

Comment:
@JayVora-SerpentCS, the field *failed** is readonly when payment has state as done due to https://github.com/odoo/odoo/blob/8.0/addons/account_payment/account_payment.py#L98
Please get it editable always, in every state. This could be done by a dedicated button for example.